tests/test_snapshot_restoration.py

```
@@ -0,0 +1,271 @@
+        f"/{index_name}/_search",
+        json={"query": [101, 201]},
+    )
+    assert req.status_code == 200, req.content
+    source_search_result = req.json() if req.content else {"results": []}
+    assert len(source_search_result["results"]) > 0
+    assert source_search_result["results"][0]["id"] == 1
+    
+    # Get attribute to verify attributes work
+    req = client.get(f"/{index_name}/_health")
+    assert req.status_code == 200, req.content
+    source_health = req.json()
```

Ruff (0.12.2)

Local variable `source_health` is assigned to but never used

Remove assignment to unused variable `source_health`

(F841)